docs/questions-and-answers.md

```diff
@@ -10,10 +12,76 @@ An Agent Service is an Autonomous Service which is implemented as a multi-agent
+In the same way companies like Apple or Google offer SDKs to accelerate devs work plus an app store to monetize their work, Autonolas offers the same capabilities but in a decentralized way: developers register components, operators run services that use those components, consumers use and pay for those services so both developers and operators are compensated for their work. And all the parameters that govern the network can be voted on.</details>
+
+<details><summary>Can services use other services?</summary>
+Yes, agent services can be composed from other agent services eventually, analogously to microservices. Sub-services could deliver all sorts of results which are consumed by a higher level service to create a higher level outcome.
+</details>
+
+<details><summary>How do services communicate with other services?</summary>
+Services can expose REST APIs and they also have a native message protocol that uses protobuf that allows them to have arbitrary message based communication between compatible agents in the network. This network is called agent communication network (ACN). When a service need a more complicated message flow than request-response (e.g. some extended dialogue like FIPA) they can express it as a protocol and deliver the messages via ACN.</p>
+Under the hood ACN is a DHT that keeps track of live agents mapping their crypto address to IP address. So agents can communicate with other agents without knowing their network location assuming they’re online or offline but registered in the ACN.
+</details>
+
+<details><summary>What happens when agents are deployed now?</summary>
+Currently we're only operating what we call "island deployments", which are services that run as one-off services, not anchored in the protocol, because the protocol isn't live (more on that here). Once the protocol is live, agents will be able to interact with it so they can monetize their work and connect to other services.
+</details>
+
+<details><summary>How many composition levels does Autonolas stack offer?</summary>
+Composition starts at the component level of the agents (multiple rounds make a skill), then continues on agent level (multiple skills make an agent) and ends at service level (multiple agents make a service).
 </details>
+
+<details><summary>How do agents settle a transaction?</summary>
+
+<ol>
+<li>Negotiation happens through ACN (or alternatively another connection like ABCI connection).</li>
+<li> A threshold of agents agree on a transaction hash.</li>
+<li>One of the agents is randomly selected as the keeper using a verifiable randomness function (currently DRAND).</li>
+<li>All agents sign the transaction using a multi-sig like Gnosis Safe.</li>
+<li>The keeper collects all the signatures and sends the transaction onchain.</li>
+<li>All agents wait for the transaction to be mined and validate the output.</li>
+<li>Done</li>
+</ol>
+</details>
+
+## Security
+
+<details><summary>How are Autonolas services run?</summary>
```

Comment:
Autonolas...

Reply:
YEs agree here the use of Autonolas is very unnecessary and it should be referred to as an agent service!